### ISSUE TITLE
RHEL 7 on Hyper-V does not configure the network

### ISSUE THREAD

**Issue by grandmasterv** [lavabit/robox#143]
I'm having a problem with the RHEL 7 boxes and Hyper-V.  The network does not seem to be configuring correctly.  Any idea if I'm doing something wrong?  

Windows 10 Enterprise v1909

**Vagrant File**
```
Vagrant.configure("2") do |config|
  config.vm.box = "generic/rhel7"
end
```

**Error**
```
    default: Configuring the VM...
==> default: Starting the machine...
==> default: Waiting for the machine to report its IP address...
    default: Timeout: 120 seconds
Hyper-V failed to determine your machine's IP address within the
configured timeout. Please verify the machine properly booted and
the network works. To do this, open the Hyper-V manager, find your
virtual machine, and connect to it.

The most common cause for this error is that the running virtual
machine doesn't have the latest Hyper-V integration drivers. Please
research for your operating system how to install these in order
for the VM to properly communicate its IP address to Hyper-V.
```

Thanks for providing these boxes to the community!

**Comment by abbbi** [comment]
i have never used vagrant with hyperv but the following statement from the error message seems legit:

```
The most common cause for this error is that the running virtual
machine doesn't have the latest Hyper-V integration drivers.
```

please login to the virtual machine you deployed via hyperv console and check if it detected any network card. Most probably rhel7 does not come with the needed drivers. Then you would have to configure your Vagrantfile to use another network card emulation, if that is possible on Hyperv. I dont know.

**Comment by ladar** [comment]
The HyperV drivers are being installed. And I've tested the config with a Windows 10 host. My guess is that you don't have a virtual switch setup properly and/or the network device isn't being configured correctly, possibly because it's using different virtual hardware than when the system was installed. 

If you login via the console, user vagrant, pw vagrant, and run `ifconfig` do you see a NIC, and does it have an IP address?

**Comment by grandmasterv** [comment]
Hi Ladar,
Thanks for checking this out. Sadly I had to move on and am using VMware now (can’t run hyperv at same time) so I can’t test this out.  I did nothing special when I set it up.  I installed hyperv via windows features.  Rebooted.  Confirmed Hyperv was installed and working.  Then I tried to run the RHEL 7 Vagrant box and got that error.  I do remember using the hyperv console to investigate but am unsure if I saw an IP or not.  I don't think I did though.  Was there some specific configuration I needed to do with HyperV before running the Vagrant box?
